frontend/js/pages/BattleList.js

```diff
@@ -27,6 +27,10 @@ function BattleList(props) {
       </div>
     );
   }
+  const battleArray = Object.values(battles.entities.battle);
+  const pokemons = battles.entities.pokemon;
+  const battleUsers = battles.entities.user;
```

Comment:
Note, that you don't need to do this inside the component, you should do this inside `mapStateToProps`

Reply:
Also, while you are there, you can denormalize your data to make the component life easier and avoid names such as `battleUsers` : https://github.com/paularmstrong/normalizr/blob/master/docs/api.md#denormalizeinput-schema-entities

Reply:
Alright! :)